### DISCUSSION TITLE
How to "Reset" a workflow after completion

### DISCUSSION THREAD

**Discussion by riaanmarx** [winder/Universal-G-Code-Sender#1658] [Help]
Is there a way to reset file(s) to <not finished> when using the Workflow plugin?
The only way I could find so far is removing the file and then adding it again.

**Comment by breiler** [comment]
If that doesn't happen its probably a bug or chance for improvement.

**Reply by riaanmarx** [reply]
After the files complete, this is how the page looks:
![image](https://user-images.githubusercontent.com/29224692/129312980-afd01d4a-90f6-4650-94cf-b9a1030e938a.png)
After clicking the ok button the check boxes stay checked, and they can not be cleared by clicking on them. Nor is there a button to clear all. I'd think a button to clear them all, or allowing the user to clear them would be best. So probably an improvement for a future release.
